src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -152,6 +158,16 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
         }
     }
 
+    public getControllerForConnection(
+        connection: KernelConnectionMetadata,
+        notebookType: typeof JupyterNotebookView | typeof InteractiveWindowView
+    ) {
+        const id =
+            notebookType === 'jupyter-notebook'
```

Comment:
can you just use the JupyterNotebookView constant here?

Reply:
Not required, as this s a strongly typed string value, hence there's no way to get a typo here.
i.e. if we make a typo here or change the constant then tsc compiler will fall over.